### ISSUE TITLE
Input field in alerts

### ISSUE THREAD

**Issue by danielmavila** [t4t5/sweetalert#351]
My alerts have been rendering with a random `fieldset` in the html, I am not using type "input" for any of these. 

```
sweetAlert({
    type: 'success'
    title: 'Report Archived'
    text: 'Report has been archived."
})
```

![screen shot 2015-05-07 at 3 46 43 pm](https://cloud.githubusercontent.com/assets/1779809/7527468/5370a3ba-f4d0-11e4-8be7-853556abb5d0.png)

**Comment by TomTasche** [comment]
check the element in your browser's developer tools. you probably have a stylesheet saying "input { display: block !important; }" somewhere

**Comment by adrianricardo** [comment]
I had a similar issue caused by foundation css rules

**Comment by danielmavila** [comment]
Hm, that might be the case for me since I am using foundation too. Thanks! 



—

**Comment by masonfox** [comment]
I am having this issue as well. It seems to be coming from the Foundation CSS file. I went through commenting things in and out and still couldn't find a solution.

If anyone has a fix, it would be much appreciated.

**Comment by TomTasche** [comment]
Assign a customClass to the dialog and then insert a css-file with the following rule at the end of your html-body:
.custom-dialog-class input {
  display: none;
}

**Comment by dlombardi** [comment]
so whats the fix here?

**Comment by TomTasche** [comment]
https://github.com/t4t5/sweetalert/issues/351#issuecomment-138983039

**Comment by ElranZiggi** [comment]
Throw this css in

.sweet-alert fieldset input {
    display: none;
}

**Comment by mbarwick83** [comment]
The fix should really be in SweetAlert's main CSS, no? .sweet-alert input {display:none!IMPORTANT}

**Comment by beastybeast** [comment]
Related to https://github.com/t4t5/sweetalert/issues/534

**Comment by t4t5** [comment]
Fixed in 2.0, thanks to the fact that each modal now renders from scratch instead.
